src/App.tsx

```diff
@@ -14,16 +14,18 @@ import Footer from './components/Layout/Footer';
 import BlogPage from './pages/blog';
 import NewsPage from './pages/news';
 import TermsOfUsePage from './pages/terms';
+import Navigation from './components/Navigation/Navigation';
 
-function App() {
+const App = () => {
   return (
     <Router>
       <GlobalProvider>
         <div className="et-app">
           <Header />
-          <div className="et-body">
+          <Navigation />
+          <div className="et-body" id="et-body">
             <div className="et-body__center">
-              <Sidebar />
+              {/* <Sidebar /> */}
```

Comment:
It seems you forgot the comment here.

Reply:
Let's keep this component comment here to keep in mind that we need a sidebar in future.